index.html

```diff
@@ -213,12 +217,31 @@ <h5>Open API + Stoplight</h5>
                     Openapi PSR7 Validator
                   </a>
                 </li>
+              <li>
+                  <a hre="https://swagger.io/specification/" target="_blank"
+                  >OpenAPI 3.0 Specs</a>
+              </li>
               </ul>
             </p>
+              <aside class="notes">
+                  <ul>
+                      <li>Come strumento per realizzare la parte di UI usiamo il design by contract (rif. slide ale maggio 2021)</li>
```

Comment:
Should we put a link to it? https://alessandrolai.dev/talks/2021-sfday-api-contracts-openapi-during-development/